spec/rails_helper.rb

```diff
@@ -181,4 +181,6 @@ def jobs
   config.after(:all) do
     FileUtils.rm_rf(Dir[Rails.root.join('spec', 'support', 'uploads')]) if Rails.env.test?
   end
+
+  config.fuubar_auto_refresh = false
```

Comment:
makes `binding.pry` work again

Reply:
There's another PR that might help this too https://github.com/department-of-veterans-affairs/vets-api/pull/3496